Britannia/events/BTWK_storyline_events.txt

```diff
@@ -4,96 +4,676 @@
 
+### PART 10 - THE FIRES OF MAI DUN ###
 
-############## UTHER'S DEATH and ARTHUR'S RETURN ##############
+### PART 11 - MYNYDD BADDON ###
 
+### PART 12 - NIMUE'S CURSE ###
 
-BTWK_story_event.0001 = {  # Triggered by on_game_start on_action.
+### PART 13 - THE LAST ENCHANTMENT ###
+
+
+############## UTHER'S ILLNESS and ARTHUR'S RETURN ##############
+
+BTWK_story_event.0101 = {
 	type = character_event
-	title = BTWK_story_event.0001.t
-	desc = BTWK_story_event.0001.desc
-	# More fitting music cue/sounds needed
-	theme = skull
-	# Won't use "window = fullscreen_event" until figure out how to add portraits to the event.
+	title = BTWK_story_event.0101.t
+	desc = BTWK_story_event.0101.desc
+	theme = realm 
+
+	right_portrait = {
+		character = character:1  # Uther
+		animation = personality_honorable
+	}	
+
+	option = {
+		name = BTWK_story_event.0101.a
+		character:1 = {
+			trigger_event = {
+				id = BTWK_story_event.0103
+				days = 50
+			}
+		}
+
+		every_ruler = {
+			if = {
+				limit = {
+					AND = {
+						holds_any_storyline_britannia_kingdoms = yes 
+						culture = {
+							culture_is_brythonic_culture_trigger = yes
+						}
+					}
+				}
+				trigger_event = {
+					id = BTWK_story_event.0102
+					days = 25
+				}
+			}
+		}
+
+	}
+}
+
+BTWK_story_event.0102 = { # Event for ai selection of whether they want to go to glevum + catalogue norwenna's suitors
+
+# MAY WANT A SECOND GLOBAL VARIABLE LIST FOR ALL VALID KINGS, IN CASE WE MAKE GLEVUM INTO A RESKINNED FEAST ACTIVITY
+# IN THIS CASE, PROBABLY WANT TO REFERENCE NEW FILE SO I DON'T HAVE TO KEEP CHANGING EVENT ID - "BTWK_storyline_activity_events", etc.
+# AI CHANCE MUST STILL BE MODIFIED ACCORDINGLY
+# ADD AN OPINION BASED MODIFIER TO AI CHANCE?
+
+	type = character_event
+	title = BTWK_story_event.0102.t
+	desc = BTWK_story_event.0102.desc
+	theme = realm 
+
+	right_portrait = {
+		character = root  
+		animation = personality_honorable
+	}
+	
+	option = {  # Go to the council
+		name = BTWK_story_event.0102.a
+		if = {
+			limit = {
+				root = {
+					is_married = no
+				}
+			}
+			add_to_global_variable_list = {
+				name = char_is_norwenna_wedding_prospect
+				target = root
+			}
+		}
+
+		else_if = {
+			limit = {
+				exists = root.primary_heir
+				root.primary_heir = {
+					is_adult = yes
+					is_married = no
+				}
+			}
+			add_to_global_variable_list = {
+				name = char_is_norwenna_wedding_prospect
+				target = root.primary_heir
+			}
+		}		
+
+		ai_chance = {
+			base = 50
+			modifier = {
+				add = 10
+				has_trait = deceitful
+			}
+			modifier = {
+				add = -10
+				has_trait = arrogant
+			}
+			modifier = {
+				add = -10
+				has_trait = vengeful
+			}
+			modifier = {
+				add = 20
+				has_trait = ambitious
+			}
+		}
+	}
+
+	option = {  # Don't go to the council
+		name = BTWK_story_event.0102.b
+		trigger = {
+			NOT = {root.primary_title = title:k_gwent} #The council is being held in Gwent at Uther's demand, they can't NOT show up? 
```

Comment:
Eh I'm sure they boycott if they really wanted (not that the AI should ever do so of course)